deploy/helm/self-service-agent/templates/employee-info-service.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.employeeInfo.enabled }}
```

Comment:
I think that the pattern here - https://github.com/rh-ai-kickstart/ai-architecture-charts/blob/main/mcp-servers/helm/templates/service.yaml might let us have one generic service.yaml verus having to have one for each mcp server we deploy.

@TomerG2 could you confirm with @yuvalturg and if so we should adopt that pattern to be consistent with the architecture charts patter, as well as cutting down on the files we'll need.